templates/admin/forms/list.html.twig

```diff
@@ -0,0 +1,154 @@
+                <li role="presentation" class="active">
+                    <a href="#registered"
+                       aria-controls="home" role="tab"
+                       data-toggle="tab">
+                        {{ "Registered"|translate }}
+                        &nbsp;<span class="badge">{{ registered|length }}</span>
+                    </a>
+                </li>
+                <li role="presentation">
+                    <a href="#unregistered"
+                       aria-controls="profile" role="tab"
+                       data-toggle="tab">
+                        {{ "Unregistered"|translate }}
+                        &nbsp;<span class="badge">{% if unregistered|length > 0 %}{{ unregistered|length }}{% endif %}</span>
+                    </a>
+                </li>
+            </ul>
+
+            <div class="tab-content">
+                <div role="tabpanel" class="tab-pane active" id="registered">
+                    <table class="table table-striped table-condensed">
+                        <thead>
+                        <tr>
+                            <th>&nbsp;</th>
+                            <th>Name</th>
+                            <th>Status</th>
+                            <th colspan="2">&nbsp;</th>
+                            <th>{{ "Priority"|translate }}</th>
+                            <th>{{ "Category"|translate }}</th>
+                            <th>{{ "Nickname"|translate }}</th>
+                            <th>{{ "Access Control"|translate }}</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% for row in registered %}
+                            <tr>
+                                <td class="text-muted"><small>{{ row.id }}</small></td>
+                                <td><strong>{{ row.name }}</strong></td>
+                                <td class="{{ row.statusColor }}">
+                                    <i class="{{ row.statusIcon }}"></i>&nbsp;
+                                    {% if row.status == "Registered" %}
+                                        {{ row.status }}
+                                    {% else %}
+                                        <a class="text-primary" href="./forms_admin.php?id={{ row.id }}&method={{ row.statusMethod }}">{{ row.status }}</a>
+                                    {% endif %}
+                                </td>
+                                <td>{{ row.packageStatus }}</td>
+                                <td class="{{ row.dbStatusClass }}">
+                                    {% if row.dbStatus == "DB Installed" %}
+                                        {{ row.dbStatus }}
+                                    {% else %}
+                                        <i class="fa fa-fw fa-database"></i>&nbsp;<a class="text-danger" href="./forms_admin.php?id={{ row.id }}&method=install_db">{{ row.dbStatus }}</a>
```

Comment:
here's a good example, where wouldn't want to use html escape. Instead should use html_attr or maybe url (although I don't think url will properly escape for html_attr since this is within an html tag).